sdk/sdk-api/src/main/java/mil/emp3/api/MilStdSymbol.java

```diff
@@ -762,14 +819,7 @@ public void setModifier(IGeoMilSymbol.Modifier eModifier, int iIndex , float fVa
      * @return A String value or null if it does not exists.
      */
     public String getStringModifier(IGeoMilSymbol.Modifier eModifier) {
-        String sValue = null;
-        java.util.HashMap<IGeoMilSymbol.Modifier, String> oModifierList =
-                this.getModifiers();
-
-        if (oModifierList.containsKey(eModifier)) {
-            sValue = oModifierList.get(eModifier);
-        }
-        return sValue;
+        return getModifiers().get(eModifier);
```

Comment:
Is the list of modifiers dynamic?  If it is static, could we make it a class variable and eliminate a few `getModifiers()` calls?

Reply:
Modifiers do appear to be dynamically set, disregard :)